src/infrastructure/figma/transformers/figma-file-transformer.ts

```diff
@@ -37,6 +40,8 @@ export const transformFileToAtlassianDesign = ({
 		status: AtlassianDesignStatus.NONE,
 		type: AtlassianDesignType.FILE,
 		lastUpdated: fileResponse.lastModified,
+		lastUpdatedBy: fileMetaResponse.file.last_touched_by,
+		iconUrl: getResourceIconUrl(),
```

Comment:
**question:** @shadyrachid , please, consider ingesting `null` by default and fallback to [JiraDesignInfoProvider.logoUrl](https://github.com/atlassian-labs/figma-for-jira/blob/59eca2b05f19be69ecf21598bc9b8ad986a3eb71/src/atlassian-connect.ts#L119) or internal Atlassian icon **on read** within Atlassian systems for the following reasons:

- `getResourceIconUrl` returns a URL to the icon from app static resources. This URL is internal to the app and can be changed at any time due to refactoring/restructuring. The app should not implicitly commit to maintain this URL over time -- it commits only to providing an icon URL through `jiraDesignInfoProvider.logoUrl`, which can use a different URL at different points of time.
- Any stored URL can get invalid at some point. Falling back to using `jiraDesignInfoProvider.logoUrl` on read allows to show the **current relevant** provider icon.